pkg/apis/core/validation/validation_test.go

```diff
@@ -1420,7 +1420,7 @@ func TestValidatePersistentVolumeClaimUpdate(t *testing.T) {
 		toggleBlockVolumeFeature(scenario.enableBlock, t)
 		scenario.oldClaim.ResourceVersion = "1"
 		scenario.newClaim.ResourceVersion = "1"
-		errs := ValidatePersistentVolumeClaimUpdate(scenario.newClaim, scenario.oldClaim)
```

Comment:
Ref:
https://github.com/kubernetes/kubernetes/blob/e7ad6e60081887d67b987cb0429b52c5a4ff878c/pkg/apis/core/validation/validation.go#L1819

newClaim is updated by function PersistentVolumeClaimUpdate(). So I passed the copied one into it. So test cases will not effect each other.